apps/backoffice-v2/src/domains/business-reports/query-keys.ts

```diff
@@ -9,22 +9,23 @@ import { TBusinessReportType } from '@/domains/business-reports/types';
 
 export const businessReportsQueryKey = createQueryKeys('business-reports', {
   list: ({
-    reportType,
```

Comment:
Why was report type removed?

Reply:
Together with `search`, they don't get "special treatment" and will be used when spreading `...params` just a couple of lines later

Reply:
No need to revert your change since we don't know when it would happen, but this query is supposed to be used for different views i.e. an ongoing reports view. Didn't understand why it was actively removed.

Reply:
It was removed from the destructure of `params` which is later on spread (`...params`) into the same place, i.e., nothing changed in the implementation. `reportType` is still being sent